contracts/OlasMech.sol

```diff
@@ -38,8 +38,10 @@ abstract contract OlasMech is Mech, IErrorsMech, ImmutableStorage {
     // Mech marketplace address
     address public immutable mechMarketplace;
     // Mech payment type
+    // TODO - can it be id or string so its' forward compatible?
     PaymentType public immutable paymentType;
 
+    // TODO give it a better name
```

Comment:
@DavidMinarsch Please provide a name, as it will require a significant change across all the contracts.

Reply:
yes still thinking